homeassistant/components/nissan_leaf/switch.py

```diff
@@ -12,9 +12,6 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Nissan Leaf switch platform setup."""
-    _LOGGER.debug(
-        "In switch setup platform, discovery_info=%s", discovery_info)
-
     devices = []
```

Comment:
Add a guard clause here that checks if discovery_info is None and return if so.

Reply:
This code is no longer present, because the "In switch setup platform... " debug log message has been removed.